src/test/java/com/yelp/nrtsearch/server/grpc/QueryTest.java

```diff
@@ -911,6 +922,22 @@ private void testWithBoost(Query originalQuery, SearchResponse searchResponse) {
     }
   }
 
+  private void testWithExplain(Query query, SearchResponse searchResponse) {
```

Comment:
It seems like this method is not called anywhere. It might be better to make this a `@Test` method for explain.